pallets/msa/src/tests.rs

```diff
@@ -591,8 +619,9 @@ pub fn create_sponsored_account_with_delegation_with_valid_input_should_succeed(
 		assert_eq!(provider_info.is_some(), true);
 
 		let events_occured = System::events();
-		let created_event = &events_occured.as_slice()[1];
-		let provider_event = &events_occured.as_slice()[2];
+		// let provider_registered_event = &events_occured.as_slice()[1];
```

Comment:
Can this comment be removed?

Reply:
It could but I think it's better if it's there.

Reply:
I can remove it but it’s documenting the event in the array which could prove useful for future modifications to that code.

Reply:
That makes sense, okay I like that